sources/packages/web/src/main.ts

```diff
@@ -1,4 +1,5 @@
 import "@bcgov/bc-sans/css/BCSans.css";
+import "@mdi/font/css/materialdesignicons.css";
```

Comment:
What is the need to add the material design icon css? Let me know if I am missing something.